config/makeCoaddTempExp_goodSeeing.py

```diff
@@ -0,0 +1,22 @@
+from lsst.decam.hits import MaxPsfWcsSelectImagesTask
+
+config.bgSubtracted=True
+
+# changing coaddName from deep requires modifying the obs_ package--wait 
+# until after upcoming RFD
+#config.coaddName='goodSeeing'
+
+config.select.retarget(MaxPsfWcsSelectImagesTask)
+config.doPsfMatch=True
+# !! This must be the same as the kernelSize in the processEimage psfMeasurement.
+config.modelPsf.size=25
```

Comment:
Can the matching constraint be enforced in code (e.g., `config.modelPsf.size=processEimage.config.kernelSize`)?

Reply:
I actually think the right thing to do here is to remove this line entirely.  `processEimage` is from `obs_lsstSim`, and it doesn't appear to me that `makeCoaddTempExp` actually uses the `size` parameter anymore.  According to the docs, `defaultFwhm` is the parameter that is used, and it needs to be tuned to the individual dataset.  I had already been passing `defaultFwhm` from the command line.